api/README.md

```diff
@@ -0,0 +1,5 @@
+# API
+
+The API will be here.
```

Comment:
Not sure to understand, is this really needed?

Reply:
Update from the distribution